### DISCUSSION TITLE
api for FALCON FUSION

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#697] [Ideas]
Greetings,

I have 2 questions

I want to know if there is an api for FALCON FUSION?

If there is no such api can I collaborate with you in order to write one?

tnx,

nate

**Comment by crowdstrikedcs** [comment]
Hi @NSH531 

Thanks for the question, at the moment there is not an API available to interact with Falcon Fusion. I'm interested in your use case, could you let us know a bit more about what you'd like to achieve with such an API?

Best,
Dixon

**Reply by NSH531** [reply]
Hi @crowdstrikedcs ,
I would love to get an API that has all the capabilities of falcon fusion.
 P.S. I am willing to help develop the required API using restful tools like django (Python).

nate

**Reply by crowdstrikedcs** [reply]
What sort of functionality are you looking for? Under the hood Fusion uses a lot of the same APIs that we expose publicly so we could investigate trying to migrate some of that functionality.

**Comment by NSH531** [comment]
II am looking for a comprehensive API on all falcon fusion functionality
nate

**Reply by crowdstrikedcs** [reply]
Hi Nate,

Since Falcon Fusion is made to allow users to make API calls without programming I'm not sure there will ever be an API to interact with it. If you give some example use cases of what you are looking to accomplish using Falcon Fusion we can maybe adapt these to FalconPy.

For example Falcon Fusion has the ability to use Real Time Response to get files after a detection. This same functionality could be implemented in FalconPy by using the Detects and Real Time Response service collections.
